### PR TITLE
Add missing `.generate-reports.py` file

### LINKED ISSUE #223: generate reports script missing
**Describe the bug**

The `.generate-reports.py` script is missing.

**To Reproduce**

Set up a project using qube.

**Expected behavior**

The workflow for report generation runs through.

[File to add ](https://github.com/qbicsoftware/cookiecutter-templates-cli/blob/master/common-files/%7B%7B%20cookiecutter.artifact_id%20%7D%7D/.generate-reports.py)

### PR DESCRIPTION
**Pull request by KochTobi**
This PR adds a missing `.generate-reports.py` script to the common files. This script is used in one of the GitHub workflows.

### CODE REVIEW

**Comment by Zethson** [comment]
@KochTobi you can either add the script to be ignored by flake8 (https://github.com/qbicsoftware/qube-cli/blob/master/setup.cfg) or run https://pypi.org/project/autopep8/ on it.

File: setup.cfg [hunk 1]
```diff
@@ -10,5 +10,5 @@ max-line-length = 160
 test = pytest
 
 [tool:pytest]
-collect_ignore = ['setup.py']
+collect_ignore = ['setup.py','.generate-reports.py']
```
Comment (Zethson) [inline setup.cfg:13]: Not here.

See my comment above.

**Comment by KochTobi** [comment]
> @KochTobi you can either add the script to be ignored by flake8 (https://github.com/qbicsoftware/qube-cli/blob/master/setup.cfg) or run https://pypi.org/project/autopep8/ on it.

Thanks @Zethson. I added it to the ignored files since it is copied from the old cookiecutter template and is intended as is to be run. Refactoring the code style can still be done later on if desired.

File: setup.cfg (hunk 1 above)
Comment (KochTobi) [reply setup.cfg:13]: oh

**Comment by Zethson** [comment]
@KochTobi I think that it needs to be under the flake8 section. (exclude)

You put it to the pytest section, which is something different.

File: setup.cfg (hunk 1 above)
Comment (Zethson) [reply setup.cfg:13]: https://github.com/qbicsoftware/qube-cli/blob/master/setup.cfg#L5

I guess

Comment (Zethson) [reply setup.cfg]: You can always try it locally before pushing by the way.

`pip install flake8`

Then just run `flake8` from the top level.

**Comment by Zethson** [comment]
@KochTobi there we go

**Comment by Zethson** [comment]
@KochTobi if you want to release this as a hotfix please also bump the version and update the changelog.

cookietemple is now on PyPI

-> `pip install cookietemple`
-> `cookietemple bump-version 2.5.2 .`

**Review by sven1103 — APPROVED** [review]

**Comment by sven1103** [comment]
@Zethson thank you for your help here!
@KochTobi thank you for adding the missing file and incorporate the necessary settings in qube

**Pull request merged**